### ISSUE TITLE
ODRViewer: LaneOffset applied to all elements

### ISSUE THREAD

**Issue by mussandreasMVIS** [esmini/esmini#490]
Hi,
I found an issue with the laneOffset.
It seems that laneOffset is applied to all elements of the road (in my example the road objects), not only the lanes.
In my screenshots you can see two roads, connected to each other. One has a laneOffset record and the other one hasn't:
![laneOffset](https://github.com/esmini/esmini/assets/150109035/4bb9890b-3f1f-441a-8672-af125e9cc79d)
This is how the same scene looks without laneOffset:
![no_laneOffset](https://github.com/esmini/esmini/assets/150109035/095c2427-3afc-4b56-9ec0-f9b730caece4)

**Comment by eknabevcc** [comment]
You're right. And this is a known deviation in esmini, reference line will shift as well.
See more info here: [User guide - Reference line and center lane while using laneOffset](https://esmini.github.io/#_reference_line_and_center_lane_while_using_laneoffset)

This behavior might be changed in future, but it's currently not prioritized.
